radis/tools/database.py

```diff
@@ -2497,9 +2531,34 @@ def read_conditions_file(path):
     None.
 
     """
+    # import csv
+    # with open(path, 'r') as csvfile:  # Check if comma or semicolon # Note: dialect may be given directly to read_csv ?
+    #     dialect=csv.Sniffer().sniff(csvfile.read(), delimiters=',;')
+
     df = pd.read_csv(path, float_precision="round_trip")
     # thank you https://stackoverflow.com/questions/36909368/precision-lost-while-using-read-csv-in-pandas
 
+    # Force some types
+    for boolvar in [
+        "self_absorption",
+        "db_use_cached",
+        "lvl_use_cached",
+        "include_neighbouring_lines",
+        "export_lines",
+        "export_rovib_fractions",
+        "load_energies",
+        "thermal_equilibrium",
+    ]:
+        if boolvar in df:
+            if df.dtypes[boolvar] == object:
+                from radis.misc.basics import str2bool
+
+                if verbose:
+                    print(
+                        f"Reading {path} : casting column {boolvar} from string type to boolean"
+                    )
+                df[boolvar] = df[boolvar].map(str2bool)
+
```

Comment:
Works, but limited to columns we already know of. I recently had a similar problem ("0.0" detected as True) after declaring a new column name. 
There must be a better way. 
I found this : 

`read_csv` directly has a `true_values` and `false_values` optional list arguments. We could replace with 
```
pd.read_csv(path, float_precision="round_trip", true_values=["true", "True", "1.0", "1"], false_values=["false", "False", "0.0", "0"])
```

Reply:
@CorentinGrimaldi if you can try ?